RetailCoder.VBE/Navigation/RegexSearchReplace/RegexSearchReplace.cs

```diff
@@ -35,15 +35,7 @@ public RegexSearchReplace(IVBE vbe, IParseCoordinator parser)
 
         public IEnumerable<RegexSearchResult> Search(string searchPattern, RegexSearchReplaceScope scope = RegexSearchReplaceScope.CurrentFile)
         {
-            Func<string,IEnumerable<RegexSearchResult>> searchFunc;
-            if (_search.TryGetValue(scope, out searchFunc))
-            {
-                return searchFunc.Invoke(searchPattern);
-            }
-            else
-            {
-                return new List<RegexSearchResult>();
-            }
+            return _search.TryGetValue(scope, out var searchFunc) ? searchFunc.Invoke(searchPattern) : new List<RegexSearchResult>();
```

Comment:
one could consider breaking the line before ? and : here, to get rid of vertical scrolling